cinema/serializers.py

```diff
@@ -0,0 +1,22 @@
+from rest_framework import serializers
+
+from cinema.models import Movie
+
+
+class MovieSerializer(serializers.Serializer):
+    id = serializers.IntegerField(read_only=True)
+    title = serializers.CharField(required=True)
+    description = serializers.CharField(required=False)

```

Comment:
if the description is not required it'd be better to set `blank=True, null=True` in its model